iree/turbine/kernel/wave/perf/benchmark.py

```diff
@@ -110,25 +125,48 @@ def main():
     parser.add_argument(
         "--num_iterations", type=int, default=100, help="Number of benchmark iterations"
     )
+    parser.add_argument(
+        "--fa", action="store_true", help="Benchmark FA instead of Wave, if available"
+    )
 
     args = parser.parse_args()
+    global flash_attn_enabled
+    if args.fa and not flash_attn_enabled:
+        raise ValueError("FA enabled but not installed on system!")
+
+    flash_attn_enabled = args.fa and flash_attn_enabled
 
     # Load configuration from JSON file
     with open(args.config, "r") as f:
         config = json.load(f)
 
     # Run benchmark for each configuration
     for shape_config in config["shapes"]:
-        benchmark_wave_sdpa(
-            shape_config["batch_size"],
-            shape_config["num_heads"],
-            shape_config["seq_len_q"],
-            shape_config["seq_len_k"],
-            shape_config["head_dim"],
-            args.num_warmup,
-            args.num_iterations,
-            args.output,
-        )
+        if not flash_attn_enabled:
+            benchmark_sdpa(
+                shape_config["batch_size"],
+                shape_config["num_heads"],
+                shape_config["seq_len_q"],
+                shape_config["seq_len_k"],
+                shape_config["head_dim"],
+                args.num_warmup,
+                args.num_iterations,
+                wave_nn.functional.wave_sdpa,
+                args.output,
+            )
+
+        if flash_attn_enabled:
+            benchmark_sdpa(
+                shape_config["batch_size"],
+                shape_config["num_heads"],
+                shape_config["seq_len_q"],
+                shape_config["seq_len_k"],
+                shape_config["head_dim"],
+                args.num_warmup,
+                args.num_iterations,
+                flash_attn_func,
```

Comment:
nit: `flash_attn_func if flash_attn_enabled else wave_nn.functional.wave_sdpa` instead of copypasting entire call

Reply:
thanks will fix